Darkly/DarklyConstants.m

```diff
@@ -0,0 +1,28 @@
+//
+//  Copyright © 2015 Catamorphic Co. All rights reserved.
+//
+
+#import "DarklyConstants.h"
+
+NSString * const kClientVersion = @"0.4.2";
+NSString * const kBaseUrl = @"https://mobile.launchdarkly.com";
```

Comment:
Mentioned in another comment that seems to have been marked outdated, but I don't think this will work. Only the events url should change to mobile-- this will change the feature flag polling as well.